lib/features/rectangles/presentation/pages/Rectangle.dart

```diff
@@ -0,0 +1,199 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+// ignore: file_names
+
+import 'dart:io';
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+import 'package:the_boxesv3/features/rectangles/domain/entities/rectiangle_image.dart';
+import 'package:the_boxesv3/features/rectangles/presentation/bloc/rectangle_imports.dart';
+import 'package:the_boxesv3/injection_container.dart';
+
+class Rect42 extends StatelessWidget {
+  final int count;
+
+  const Rect42({Key? key, required this.count}) : super(key: key);
+
+  int getCount() {
+    return count;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Rectangle Image List'),
+      ),
+      body: buildBody(),
+    );
+  }
+
+  BlocProvider<RectangleBloc> buildBody() {
+    sl.allowReassignment = true;
+    sl.registerLazySingleton(() => getCount());
+    return BlocProvider(
+      create: (_) => sl.get<RectangleBloc>()..add(const LoadRectangle()),
+      child: BlocBuilder<RectangleBloc, RectangleState>(
+        builder: (context, state) {
+          if (state is RectangleLoading) {
+            return const CircularProgressIndicator();
+          } else if (state is RectangleLoaded) {
+            return ImageListView(imageList: state.imageList);
+          } else if (state is Error) {
+            return Center(
+              child: Text(state.message, textDirection: TextDirection.ltr),
+            );
+          } else {
+            return const Center(
+                child: Text('something went wrong',
+                    textDirection: TextDirection.ltr));
+          }
+        },
+      ),
+    );
+  }
+}
+
+class ImageListView extends StatelessWidget {

```

Comment:
it's common practice to separate that into different files

Reply:
ok, I will do that.